src/htmx.js

```diff
@@ -5116,7 +5116,7 @@ var htmx = (function() {
       "[hx-trigger='restored'],[data-hx-trigger='restored']"
     )
     body.addEventListener('htmx:abort', function(evt) {
-      const target = evt.target
+      const target = (/** @type {CustomEvent} */(evt)).detail.elt || evt.target
```

Comment:
should we try evt.target first?  This seems like it could break behavior people that were using `detail.elt` for other reasons as it is